cmd/porter/bundle.go

```diff
@@ -32,11 +32,18 @@ func buildBundleCommands(p *porter.Porter) *cobra.Command {
 
 func buildBundleCreateCommand(p *porter.Porter) *cobra.Command {
 	return &cobra.Command{
-		Use:   "create",
+		Use:   "create [bundle-name]",
 		Short: "Create a bundle",
-		Long:  "Create a bundle. This generates a porter bundle in the current directory.",
+		Long:  "Create a bundle. This command creates a new porter bundle with the specified bundle-name, in the directory with the specified bundle-name. If no bundle-name is provided, the bundle will be created in current directory and the bundle name will be 'porter-hello'.",
+		Args:  cobra.MaximumNArgs(1), // Expect at most one argument for the bundle name
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return p.Create()
+			// By default we create the bundle in the current directory
+			bundleName := ""
+			if len(args) > 0 {
```

Comment:
If create the bundleName string then we can skip the `if len(args) > 0` 
and just do the 
```
bundleName = args[0]
```